pages/events.vue

```diff
@@ -35,8 +35,8 @@ function renderMarkdown(description: string) {
 
 const createEventsList = (events: any) => {
   return events.map((event: any) => new Event(
-    event.start.dateTime,
-    event.end.dateTime,
+    event.start.dateTime || event.start.date,
+    event.end.dateTime || event.end.date,
```

Comment:
Yep, that looks awesome! Thanks @charrde I'm going to merge a bit later today so that we can observe the deployment together.